src/twisted/mail/pop3.py

```diff
@@ -855,7 +858,13 @@ def do_PASS(self, password):
 
         @type password: L{bytes}
         @param password: A password.
+
+        @type args: L{tuple} or L{None}
+        @param args: Other words composing the password.
         """
+        if args:
+            password += ' ' + ' '.join(args)
```

Comment:
Those are also good changes to make.  And filing a new ticket to handle those changes is the right idea, yea.  These particular strings should *probably* still change since they're new in this branch (but you *could* also leave them as long as you promise to fix the whole module after this branch is merged :smile: ).